### PR TITLE
Issue#905 Add updating null values in txt file

### PR DESCRIPTION
**Pull request by MakarovYaroslav**
Two functions have been added for replacing in txt file bad values('-999.00', '-999') by 'None'.

### CODE REVIEW

**Comment by henrykironde** [comment]
@MakarovYaroslav thanks for the contribution. I do like what you have done with the code 👍.
On the other hand, the retriever has the capabilities of handling that internally.

Sorry the issue was not clear, but this can be done with identifying the null values used in this dataset and adding them to the list in the part shown below.

```
            self.cleanup_func_table = Cleanup(correct_invalid_value, nulls=['NA', 'more-values'])
        else:
            self.cleanup_func_table = Cleanup(correct_invalid_value, missing_values=['NA', ' [more-values]'])
```
Note: 
**Why the issue was created**: The dataset seemed to use more than one value to represent null values, `NA` being one of them. This is based on the last time I went through the dataset.

**Comment by henrykironde** [comment]
Looks good just a few changes.  set the version number to `1.3.2`
run `Python version.py ` to update the `version.txt` and commit all changes

**Comment by MakarovYaroslav** [comment]
Done. Thanks for explanation @henrysenyondo. Can you reply to my message in gitter, please?

**Comment by henrykironde** [comment]
@MakarovYaroslavm sorry for the late reply. I read it and forgot about it. Thanks for the reminder.

File: version.txt [hunk 1]
```diff
@@ -53,7 +54,7 @@ nematode_traits.json,1.0.0
 ngreatplains-flowering-dates.json,1.0.0
 npn.py,2.1.2
 nyc_tree_count.json,1.0.0
-pantheria.py,1.3.1
+pantheria.py,1.3.2
```
Comment (henrykironde) [inline version.txt]: Some info should not be in this PR. Looks like you have some scripts that have been added in the version.txt.
Clean up the repo and run python version.py again.

You should also squash the commits to one. 

```
git clean -fx -d
retriever reset scripts
python version.txt
```

**Comment by henrykironde** [comment]
@MakarovYaroslav, all looks good. Thanks for checking those values

**Pull request merged**